source/include/JavaScriptRef.h

```diff
@@ -0,0 +1,42 @@
+/**
+
+ Copyright (c) 2018 National Instruments Corp.
+
+ This software is subject to the terms described in the LICENSE.TXT file
+
+ SDG
+ */
+
+/*! \file
+ \brief JavaScript refnum
+
+ JavaScript references are defined in the Params or Locals section of a VI,
+ and define a named, non-statically-bound reference to a JavaScript opaque object.
+ 
+ Example:
+ 
```

Comment:
Doxygen supports Markdown so you can specify a code block for this snippet by adding spaces https://www.stack.nl/~dimitri/doxygen/manual/markdown.html#md_codeblock